dev/tasks/linux-packages/debian.ubuntu-xenial/rules

```diff
@@ -19,16 +19,28 @@ override_dh_autoreconf:
 	  ./autogen.sh
 
 override_dh_auto_configure:
+	if dpkg -l nvidia-cuda-toolkit > /dev/null 2>&1; then	\
+	  ARROW_CUDA=ON;					\
+	  ARROW_PLASMA=ON;					\
+	else							\
+	  ARROW_CUDA=OFF;					\
+	  ARROW_PLASMA=OFF;					\
+	fi;							\
+	if dpkg -l clang-7 > /dev/null 2>&1; then		\
```

Comment:
I think you want to check for `llvm-7-dev` instead.

Reply:
I just copied it from "linux-packages/debian/rules" file. And "apache-arrow" build-depends on "clang-7".

Reply:
actually, my previous comment was wrong, gandiva requires clang since it uses clang to compile a tiny c library to binary IR.